stdlib/universe/stddev.go

```diff
@@ -67,7 +67,8 @@ func (s *StddevProcedureSpec) Copy() plan.ProcedureSpec {
 }
 
 type StddevAgg struct {
-	n, m2, mean float64
+	n        int64
```

Comment:
Why was `n` changed to an `int64` I do not see any uses of `n` as an `int64`? It seems to be being converted to a `float64` on each use.

Reply:
I thought it was better to help maintain accuracy during comparisons. The two are functionally  the same so I figured keeping it as the proper type (an integer for counting) would reduce potential errors.

Reply:
I have reverted it back.